src/Tree/Tree.cpp

```diff
@@ -83,11 +83,17 @@ int Tree<ndim,ParticleType,TreeCell>::ComputeActiveParticleList
 //=================================================================================================
 template <int ndim, template<int> class ParticleType, template<int> class TreeCell>
 int Tree<ndim,ParticleType,TreeCell>::ComputeActiveCellList
- (TreeCell<ndim> *celllist)            ///< Array containing copies of cells with active ptcls
+ (TreeCell<ndim>** celllistaux)            ///< Array containing copies of cells with active ptcls
```

Comment:
I'm not keen on a function that allocated memory and leaves someone else to clean up after it.  It's begging for someone to create a memory leak. Can we not instead use vectors here?